src/reference/asciidoc/whats-new.adoc

```diff
@@ -85,3 +85,8 @@ A `RotatingServerAdvice` is now available to poll multiple servers and/or direct
 See <<ftp-rotating-server-advice>> and <<sftp-rotating-server-advice>> for more information.
 
 Also inbound adapter `localFilenameExpression` s can contain the variable `#remoteDirectory` which contains the remote directory being polled.
+
+The generic type of the comparators, used to sort the fetched file list for the streaming adapters, has changed from `AbstractFileInfo<F>` to simply `<F>`.
```

Comment:
Well, semantically an `<F>` is not equal to `F`, however it definitely feels like you are talking about exactly `F`. Just because your `AbstractFileInfo<F>` is not wrapped to the angle brackets